src/Processors/Formats/Impl/JSONAsStringRowInputFormat.cpp

```diff
@@ -0,0 +1,135 @@
+#include <Processors/Formats/Impl/JSONAsStringRowInputFormat.h>
+#include <Formats/JSONEachRowUtils.h>
+#include <common/find_symbols.h>
+#include <IO/ReadHelpers.h>
+
+namespace DB
+{
+
+extern bool fileSegmentationEngineJSONEachRowImpl(ReadBuffer & in, DB::Memory<> & memory, size_t min_chunk_size);
```

Comment:
is redundant, since the declaration is in `JSONEachRowUtils.h`

Reply:
Ok, thank you!